gammapy/irf/effective_area_table.py

```diff
@@ -360,8 +364,24 @@ class EffectiveAreaTable2D(object):
     >>> from gammapy.datasets import load_aeff2D_fits_table
     >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
     >>> offset = Angle(0.6, 'degree')
-    >>> energies = Quantity(np.logspace(0, 1, 60), 'TeV')
-    >>> eff_area = aeff2D.evaluate(offset, energies)
+    >>> energy = Quantity(np.logspace(0, 1, 60), 'TeV')
+    >>> eff_area = aeff2D.evaluate(offset, energy)
+
+    Create ARF fits file for a given offest and energy binning:
+
+    >>> import numpy as np
+    >>> from astropy.coordinates import Angle
+    >>> from astropy.units import Quantity
+    >>> from gammapy.irf import EffectiveAreaTable2D
+    >>> from gammapy.datasets import load_aeff2D_fits_table
+    >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
+    >>> offset = Angle(0.43, 'degree')
+    >>> nbins = 50
+    >>> energy = Quantity(np.logspace(0, 1, nbins+1), 'TeV')
```

Comment:
ok sorry, I  was not clear (but I agree this is not the place to discuss this)
Just for completeness: I was talking about two different things
1) energy bound (nbins from emin to emax): used to generate an energy axis. I would prefer the API (emin, emax, nbins)
2) energy axis (nparray holding the axis): when passed to a function (e.g. effective area table). I would make more sense to me to pass the axis with lenght (nbins+1) instead of passing two arrays (energ_lo, enerh_hi) with upper and lower bound. If we would pass only one axis this could be generated with the energy_bounds... function, otherwise there are unnecessary lines like energ_lo = energy_axis[:-1] and ener_hi = energy_axis[1:]